### ISSUE TITLE
Google Pay Button missing on Android

### ISSUE THREAD

**Issue by felixkrautschuk** [jibon57/nativescript-braintree#32]
When running the demo app of this repo on my Android device (Google Pixel 3a Android 10), I do not see the Google Pay option, only PayPal and Credit card. I have configured Google Pay on my device correctly.

```
let opts: BrainTreeOptions = {
            amount: "0.01",
            collectDeviceData: false,
            requestThreeDSecureVerification: true,
            enableGooglePay: true,
            // Apple Pay payment request
            currencyCode: "EUR"
};
```

The meta-data tag is included in the AndroidManifest as described in the docs (it already exists within the demo app)

My config:
- nativescript cli 6.5.0
- nativescript/core 6.5.1
- tns android/ios 6.5.0
- nativescript-braintree 2.0.7

The Apple Pay Button is also missing on iOS, but that could be caused by the fact that I have not configured anything about Apple Pay on that device.

**Comment by bastienlemaitre** [comment]
Hi, same issue.. Any news ? @jibon57

**Comment by felixkrautschuk** [comment]
@bastienlemaitre I figured out what was causing this on my side.

Firstly, you have to add `<meta-data android:name="com.google.android.gms.wallet.api.enabled" android:value="true"/>` to the **application** tag within your AndroidManifest.xml (this is fixed in the demo app of version 3.0.0 of this plugin as well).

Secondly, you have to go to the control panel of your Braintree Sandbox Account -> _Settings_ -> _Processing_ -> _Payment_ _Methods_ and enable Google Pay there.

**Comment by bastienlemaitre** [comment]
Thx !
